test_gui/wiki_test.py

```diff
@@ -68,3 +68,15 @@ def test_goto_mobile_view(self):
         article_page = self.start.at(ArticlePage, substitutions=(article_to_use))
         self.is_in(article_to_use, article_page.get_title)
         mobile_page = article_page.goto_mobile_view()
+
+    @attr("Example", "Login")
+    def test_login_logout(self):
+        landing_page = self.start.at(WikiHomePage)
+        main_page = landing_page.goto_english()
+        login_page = main_page.goto_login()
+        wiki_creds = self.env.get_credentials('wiki_creds', {
```

Comment:
Don't provide a default, but add a comment about adding credentials in my_cfg.py.